### PR TITLE
Model Implementation

### PR DESCRIPTION
**Pull request by ArcaneWanderer**

### CODE REVIEW

File: src/model/Category.java [hunk 1]
```diff
@@ -3,4 +3,13 @@
 public class Category {
     public int categoryId;
     private String categoryName;
+
+    public Category(String categoryName) {
```
Comment (PSYQUO) [inline src/model/Category.java]: Is there a reason not to include Ids as constructor parameter?

File: src/model/Consumable.java [hunk 2]
```diff
@@ -8,6 +8,53 @@
     private String codeName;
     private Category category;
     private double price;
-    private List<RawItem> rawItem;
+    private List<RawItem> rawItems;
     private Meal meal;
+
+    public Consumable(String name, String codeName, Category category, double price, List<RawItem> rawItem, Meal meal) {
```
Comment (PSYQUO) [inline src/model/Consumable.java]: Same with this

File: src/model/Ingoing.java [hunk 3]
```diff
@@ -1,9 +1,30 @@
 package model;
 
-import java.time.LocalDatetime;
+import java.time.LocalDateTime;
 
 public class Ingoing {
-    private LocalDatetime inDate;
+    private LocalDateTime inDate;
```
Comment (PSYQUO) [inline src/model/Ingoing.java]: noice 👍

File: src/model/Category.java (hunk 1 above)
Comment (ArcaneWanderer) [reply src/model/Category.java]: I think when I made this, there was but I overlooked a certain problem.

Comment (ArcaneWanderer) [reply src/model/Category.java]: Editing

**Pull request merged**